ui/apps/platform/src/Containers/VulnMgmt/Entity/TableWidgetFixableCves.js

```diff
@@ -45,16 +44,15 @@ const TableWidgetFixableCves = ({
     const [cveSort, setCveSort] = useState(defaultCveSort);
 
     const { isFeatureFlagEnabled } = useFeatureFlags();
-    const showVMUpdates = isFeatureFlagEnabled('ROX_POSTGRES_DATASTORE');
 
     const displayedEntityType = resourceLabels[entityType];
     const displayedVulnType = resourceLabels[vulnType];
 
     const queryFieldName = queryFieldNames[entityType];
-    let queryVulnCounterFieldName = showVMUpdates ? 'imageVulnerabilityCounter' : 'vulnCounter';
-    let queryVulnsFieldName = showVMUpdates ? 'imageVulnerabilities' : 'vulns';
-    let queryCVEFieldsName = showVMUpdates ? 'imageCVEFields' : 'cveFields';
-    let queryFragment = showVMUpdates ? IMAGE_CVE_LIST_FRAGMENT : VULN_CVE_LIST_FRAGMENT;
+    let queryVulnCounterFieldName = 'imageVulnerabilityCounter';
+    let queryVulnsFieldName = 'imageVulnerabilities';
+    let queryCVEFieldsName = 'imageCVEFields';
+    let queryFragment = IMAGE_CVE_LIST_FRAGMENT;
     let exportType = entityTypes.CVE;
```

Comment:
I think we can completely remove the export type of `entityTypes.CVE` from this section, too.